packages/block-editor/src/components/block-list/block.js

```diff
@@ -450,6 +451,21 @@ function BlockListBlock( {
 		};
 	}
 	const blockElementId = `block-${ clientId }`;
+	const { position, orientation } = moverOptions;
```

Comment:
`position` and `orientation` might have to be given a more specific alias. In the context of `BlockListBlock` they could refer to lots of things.